src/main/java/su/litvak/chromecast/api/v2/ChromeCast.java

```diff
@@ -86,6 +86,21 @@ public String getApplication() {
     public void setApplication(String application) {
         this.application = application;
     }
+    
+    /** Returns the {@link #channel} and opens it if it's not yet or
+     * no longer open.
+     * @return an open channel.
+     */
+    private Channel channel() throws IOException {
```

Comment:
I think we should not break current behaviour. Instead of this we should add a property, something like `boolean autoReconnect`. If you set it to `true`, then channel will be re-opened when it is disconnected. It does not make much sense to call `connect` in this case. Maybe we should set it `true` by default. When it is set to `false`, then user should be responsible for recovery after connection loss.

Please let me know what do you think.